src/status_im/subs/communities_test.cljs

```diff
@@ -503,3 +504,52 @@
+  [sub-name]
+  (testing "returns sorted community members per online status"
+    (let [token-image-eth "data:image/jpeg;base64,/9j/2w"
+          community       {:id                  community-id
+                           :permissions         {:access 3}
+                           :token-images        {"ETH" token-image-eth}
+                           :name                "Community super name"
+                           :chats               {"89f98a1e-6776-4e5f-8626-8ab9f855253f"
+                                                 {:description "x"
+                                                  :emoji       "🎲"
+                                                  :permissions {:access 1}
+                                                  :color       "#88B0FF"
+                                                  :name        "random"
+                                                  :categoryID  "0c3c64e7-d56e-439b-a3fb-a946d83cb056"
+                                                  :id          "89f98a1e-6776-4e5f-8626-8ab9f855253f"
+                                                  :position    4
+                                                  :can-post?   false
+                                                  :members     {"0x04" {"roles" [1]}}}
+                                                 "a076358e-4638-470e-a3fb-584d0a542ce6"
+                                                 {:description "General channel for the community"
+                                                  :emoji       "🥔"
+                                                  :permissions {:access 1}
+                                                  :color       "#4360DF"
+                                                  :name        "general"
+                                                  :categoryID  "0c3c64e7-d56e-439b-a3fb-a946d83cb056"
+                                                  :id          "a076358e-4638-470e-a3fb-584d0a542ce6"
+                                                  :position    0
+                                                  :can-post?   false
+                                                  :members     {"0x04" {"roles" [1]}}}}
+                           :members             {"0x01" {"roles" [1]}
+                                                 "0x02" {"roles" [1]}
+                                                 "0x03" {"roles" [1]}
+                                                 "0x04" {"roles" [1]}}
+                           :can-request-access? false
+                           :outroMessage        "bla"
+                           :verified            false}]
+      (swap! rf-db/app-db assoc-in [:communities community-id] community)
+      (swap! rf-db/app-db assoc :profile/profile profile-test/sample-profile)
+      (swap! rf-db/app-db assoc
+        :visibility-status-updates
+        {"0x01" {:status-type 3}
+         "0x02" {:status-type 3}})
+      (is (= '({:title :Offline
```

Comment:
I wanted to make `clj-kondo` happy
![CleanShot 2024-01-31 at 12 31 22](https://github.com/status-im/status-mobile/assets/33176106/1f19eb20-4bcf-4dcc-bd6b-cfd60a2240a5)
And `make lint` failed
```clj
src/status_im/subs/communities_test.cljs:552:23: error: a string is not a function
src/status_im/subs/communities_test.cljs:554:23: error: a string is not a function
linting took 28129ms, errors: 2, warnings: 1218
make: *** [lint] Error 3
ibrkhalil@ibrkhalil status-mobile % 
```

Reply:
@ibrkhalil, a list can't be used as a data structure without being quoted, otherwise Clojure will try to use the first argument as a function. That's why you see the complaint.

But in tests, we use vectors for equality checks because most of the time we don't actually care if it's a vector instance or a lazy seq for example.

Reply:
Got it, Thanks Icaro